Q/QPALM/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QPALM"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Benny44/QPALM_vLADEL.git", "370ccbeb33baaa566c3de2bde5a69944994593e2"),
+    GitSource("https://github.com/Benny44/LADEL.git", "585b766da25e028f368e4d9039ef69e326baeb5b"),
```

Comment:
Why these aren't two different packages?

Reply:
I put them together because I don't believe there's a Julia interface to LADEL, and I'm not sure the QPALM build system could handle it (I haven't tried). The instructions say that the LADEL source should be in the QPALM tree (e.g., using `git submodule`).